incubator/nextcloud/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+- prydonius
+- tompizmor
+- sameersbn
+- carrodher
+- affix
+reviewers:
+- prydonius
+- tompizmor
+- sameersbn
+- carrodher
+- affix
```

Comment:
For the time you can drop this file. It looks like you copied this from a chart managed by Bitnami. Since you're not a member of the Kubernetes org at the moment this isn't going to enable you to review or merge for the chart. Once you have some more contributions you can apply for membership and then a file like this will make sense.